dist/tools/zep_dispatch/start_network.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+ZEP_DISPATCH_DIR="$(cd "$(dirname "$0")" && pwd -P)"
+UHCPD="$(cd "${ZEP_DISPATCH_DIR}/../uhcpd/bin" && pwd -P)/uhcpd"
+DHCPD="$(cd "${ZEP_DISPATCH_DIR}/../dhcpv6-pd_ia/" && pwd -P)/dhcpv6-pd_ia.py"
+ZEP_DISPATCH="${ZEP_DISPATCH_DIR}/bin/zep_dispatch"
+TAPSETUP="$(cd "${ZEP_DISPATCH_DIR}/../tapsetup/" && pwd -P)/tapsetup"
+
+TAP_GLB="fdea:dbee:f::1/64"
+
+NOSUDO="sudo -u ${SUDO_USER}"
```

Comment:
I rather would prefer a warning here. Calling `sudo` in a script, especially in such a hidden manner, seems not very nice.

Reply:
NVM, the `-u` actually "unsudos" the binay called.